forgerock-auth/build.gradle

```diff
@@ -93,7 +94,6 @@ dependencies {
 
     def coroutine_version = '1.7.2'
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$coroutine_version"
-    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutine_version"
```

Comment:
we dont need this anymore ..all the coroutines are part of `org.jetbrains.kotlinx:kotlinx-coroutines-android:`